sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.netstandard2.0.cs

```diff
@@ -120,3 +120,73 @@ public ProvisionableTemplateAttribute(string resourceName) { }
         public string ResourceName { get { throw null; } }
     }
 }
+namespace System.ClientModel
+{
+    public partial interface IAuthorizationCodeFlowContext : System.ClientModel.IScopedFlowContext, System.ClientModel.ITokenContext
+    {
+        System.Uri AuthorizationUri { get; }
+        System.Uri RefreshUri { get; }
+        System.Uri TokenUri { get; }
+    }
+    public partial interface IClientCredentialsFlowContext : System.ClientModel.IScopedFlowContext, System.ClientModel.ITokenContext
+    {
+        System.Uri RefreshUri { get; }
+        System.Uri TokenUri { get; }
+    }
+    public partial interface IImplicitFlowContext : System.ClientModel.IScopedFlowContext, System.ClientModel.ITokenContext
+    {
+        System.Uri AuthorizationUri { get; }
+        System.Uri RefreshUri { get; }
+    }
+    public partial interface IPasswordFlowContext : System.ClientModel.IScopedFlowContext, System.ClientModel.ITokenContext
+    {
+        System.Uri RefreshUri { get; }
+        System.Uri TokenUri { get; }
+    }
+    public partial interface IScopedFlowContext : System.ClientModel.ITokenContext
+    {
+        string[] Scopes { get; }
+        object CloneWithAdditionalScopes(string[] additionalScopes);
+    }
+    public partial interface ITokenContext
+    {
+    }
+    public partial interface ITokenProvider
+    {
+        object CreateContext(System.Collections.Generic.IReadOnlyDictionary<string, object> properties);
+        System.ClientModel.Token GetAccessToken(object context, System.Threading.CancellationToken cancellationToken);
+        System.Threading.Tasks.ValueTask<System.ClientModel.Token> GetAccessTokenAsync(object context, System.Threading.CancellationToken cancellationToken);
```

Comment:
Is there a reason we're using `object` instead of `ITokenContext` in these APIs?